to_csv.py

```diff
@@ -0,0 +1,49 @@
+import csv
+
+
+class ToCsv:
+    """
+    Class that create, delete, and edit csv files
+    """
+
+    def __init__(self, output_file: str):
+        """
+        initialize the path of the output file
+        :param output_file:
+        """
+        self.output_file = output_file
+        self.file_obj = open(self.output_file, 'w')
```

Comment:
This file will be left open if some exception occurs between `__init__()` and `end_session()` calls.

Reply:
Fixed.